js/index.js

```diff
@@ -38,5 +38,83 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+//let logo = document.getElementById("logo-img");
+//logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+/* -------- Images ------- */
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.src = siteContent["nav"]["img-src"];
+
+let ctaImg = document.getElementById("cta-img");
+ctaImg.src = siteContent["cta"]["img-src"];
+
+let midImg = document.getElementById("middle-img");
+midImg.src = siteContent["main-content"]["middle-img-src"];
+
+/* -------- Nav ------- */
+const mainNav = document.querySelector("nav");
+
+const navAnchors = document.querySelectorAll("a");
+
+navAnchors.forEach((index, i) => navAnchors[i].textContent = siteContent["nav"][`${Object.keys(siteContent["nav"])[i]}`]);
+
+//
+const firstAnchor = document.createElement("a");
+firstAnchor.href = "#";
+firstAnchor.textContent = "Case Studies";
+firstAnchor.style.color = "green";
+
+const lastAnchor = document.createElement("a");
+lastAnchor.href = "#";
+lastAnchor.textContent = "Blog";
+lastAnchor.style.color = "green";
+
+mainNav.prepend(firstAnchor);
+mainNav.appendChild(lastAnchor);
+
+navAnchors.forEach(navAnchor => navAnchor.style.color = "green");
+
+/* -------- CTA ------- */
+
+const ctaHeader = document.querySelector("h1");
+ctaHeader.innerHTML = siteContent["cta"]["h1"].split(' ').join(`<br>`);
+
+const ctaButton = document.querySelector("button");
+ctaButton.textContent = siteContent["cta"]["button"];
+
+/* -------- Main ------- */
+const contentHeaders = document.querySelectorAll("h4");
```

Comment:
Doing it this way here is the right play I think, but as a reminder remember any valid CSS selector works for the query selectors. That means you can do something like:

```js
let topTextContentStuff = document.querySelectorAll(".top-content .text-content h4");
```

This will target the two `h4` elements in the `.text-content` divs, which are children of `.top-content`. This allows you to be very specific in your selections without needing to modifying the original HTML with additional classes or ID's.